addons/account_facturx/models/account_move.py

```diff
@@ -54,12 +54,13 @@ def _import_facturx_invoice(self, tree):
         '''
         amount_total_import = None
 
+        default_type = False
         if self._context.get('default_journal_id'):
             journal = self.env['account.journal'].browse(self.env.context['default_journal_id'])
             default_type = 'out_invoice' if journal.type == 'sale' else 'in_invoice'
         elif self._context.get('default_type'):
             default_type = self._context['default_type']
-        else:
+        elif self.type not in self.env['account.move'].get_invoice_types(include_receipts=True):
```

Comment:
that puzzles me: I think what you want to test is 
```python
elif not self.type:
    raise UserError(_("No information about the journal or the type of invoice is passed"))
```
because that's what the error message is saying, as well as the rest of the code: 
```python
self.type = default_type or self.type
```

Reply:
```python
...
elif self.type in self.env['account.move'].get_invoice_types(include_receipts=True):
    default_type = self.type

if not default_type:
    raise UserError(_("No information about the journal or the type of invoice is passed"))

...

self.type = default_type
```